glean-core/csharp/Glean/Configuration/Configuration.cs

```diff
@@ -16,6 +16,10 @@ public sealed class Configuration
         public string serverEndpoint;
         public string channel;
         public int? maxEvents;
+        public bool logPings;
+        public string pingTag;
+
```

Comment:
It looks like you miss adding `userAgent`.

Reply:
Oh, good point. I'll add it, but we just filed a bug to consider [removing it from all the bindings](https://bugzilla.mozilla.org/show_bug.cgi?id=1646441).

Reply:
This actually led me to find that Glean has a bug currently: it's not setting the right user agent: https://bugzilla.mozilla.org/show_bug.cgi?id=1646626